knowage-core/src/main/java/it/eng/spagobi/api/AbstractDocumentResource.java

```diff
@@ -115,7 +115,7 @@ public Response getDocumentDetails(Object documentIdentifier) {
 
 	}
 
-	public Response updateDocument(String label, String body) {
+	protected Response updateDocument(String label, String body) {
```

Comment:
If possible, I would prefer private but this is an abstract class, we need a check.

Reply:
This one can't be private, it is being used in DocumentResource.java (two files, one of them is v2).